x-pack/plugins/security_solution/common/ecs/index.ts

```diff
@@ -44,10 +45,11 @@ export interface Ecs {
   event?: EventEcs;
   geo?: GeoEcs;
   host?: HostEcs;
+  kibana?: KibanaEcs;
```

Comment:
We need to get the API hooked up to ensure that these are coming back as nested objects, rather than a flattened set of key/values at the top level of the `Ecs` object... on the backend, we're handling as a flat set of key/value pairs before indexing.

Reply:
![image](https://user-images.githubusercontent.com/17211684/130612321-e36cab5a-e721-4667-8b66-0b08e62f07db.png)